packages/nimble-components/src/combobox/template.ts

```diff
@@ -34,6 +34,9 @@ ComboboxOptions
         @focusout="${(x, c) => x.focusoutHandler(c.event as FocusEvent)}"
         @keydown="${(x, c) => x.keydownHandler(c.event as KeyboardEvent)}"
     >
+        <label part="label" class="label">
```

Comment:
Same issue for select

Reply:
Looks like we aren't consistent across controls. Switch label is selectable but text area and text field labels aren't. Maybe create a bug to make consistent.

Reply:
Based on the other comment we should have the issue align our controls on mouse selection, hover interactions, and click activation.

Reply:
That issue should also probably capture some docs on labelled control micro interactions to validate are consistent.

Reply:
I attempted to capture all of this discussion / document the inconsistencies in #2331 .

I considered moving the `user-select: none` style, but left it as-is for this PR, since the current behavior matches more of the other labeled controls, and it's currently a shared dropdown style. We can revisit as part of that issue, as needed.